packages/effects-core/src/vfx-item.ts

```
@@ -581,6 +579,29 @@ export class VFXItem extends EffectsObject implements Disposable {
+    // 重新设置当前元素和组件的 ID 以及子元素和子元素组件的 ID，避免实例化新的对象时产生碰撞
+    this.resetGUID();
+    const newItem = this.engine.findObject<VFXItem>({ id: this.defination.id });
+
+    newItem.resetGUID();
+    this.resetGUID(previousObjectIDMap);
+
+    if (this.composition) {
+      newItem.setParent(this.composition.rootItem);
+    }
+
+    return newItem;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)